In [8]:
import pandas as pd

train_df = pd.read_csv("./ssec/val.csv", error_bad_lines=False, header=None, delimiter="\t")
train_df

/Users/ItishaYadav1/opt/anaconda3/lib/python3.9/site-packages/IPython/core/interactiveshell.py:3444: FutureWarning: The error_bad_lines argument has been deprecated and will be removed in a future version.


  exec(code_obj, self.user_global_ns, self.user_ns)


0             1        2     3    4        5         6      7  \
0    Anger  Anticipation  Disgust   ---  Joy      ---       ---    ---   
1      ---  Anticipation      ---   ---  Joy      ---  Surprise    ---   
2    Anger  Anticipation  Disgust  Fear  ---  Sadness       ---    ---   
3    Anger           ---      ---   ---  Joy      ---       ---  Trust   
4    Anger  Anticipation  Disgust   ---  ---  Sadness       ---    ---   
..     ...           ...      ...   ...  ...      ...       ...    ...   
287  Anger           ---  Disgust   ---  ---  Sadness       ---    ---   
288    ---           ---      ---   ---  ---  Sadness       ---  Trust   
289  Anger  Anticipation  Disgust   ---  ---  Sadness       ---    ---   
290  Anger           ---  Disgust   ---  ---  Sadness       ---    ---   
291  Anger           ---  Disgust   ---  ---      ---  Surprise    ---   

                                                     8  
0    on a side note, just because you think smtg is...  
1    Woah the hashtag puts the rainbow heart itself...  
2    I refuse 2comply w/Bad laws. Just Because #SCO...  
3    Just owned a woman who works for planned paren...  
4    If it's getting old maybe you should stop call...  
..                                                 ...  
287  There's a law protecting unborn eagles, but no...  
288  I am 1 in 3... I have had an abortion #Abortio...  
289  How dare you say my sexual preference is a cho...  
290  Equal rights for those 'born that way', no rig...  
291  #POTUS seals his legacy w/ 1/2 doz wins. The #...  

[292 rows x 9 columns]

# Data Details # Multi-label-classification
# Emotions tags:
Anger, Anticipation, Disgust, Fear, Joy, Sadness, Surprise, Trust
train.csv = 2622
test.csv = 1956
val.csv = 292

In [16]:
isear_train_df = pd.read_csv("./isear/isear-train.csv", error_bad_lines=False, header=None)
isear_train_df

/Users/ItishaYadav1/opt/anaconda3/lib/python3.9/site-packages/IPython/core/interactiveshell.py:3444: FutureWarning: The error_bad_lines argument has been deprecated and will be removed in a future version.


  exec(code_obj, self.user_global_ns, self.user_ns)
b'Skipping line 119: expected 2 fields, saw 7\nSkipping line 1213: expected 2 fields, saw 4\nSkipping line 2323: expected 2 fields, saw 3\nSkipping line 2803: expected 2 fields, saw 3\nSkipping line 3630: expected 2 fields, saw 4\nSkipping line 4635: expected 2 fields, saw 5\nSkipping line 4797: expected 2 fields, saw 4\n'


0                                                  1
0         joy  When I understood that I was admitted to the U...
1        fear  I broke a window of a neighbouring house and I...
2         joy                         Got a big fish in fishing.
3        fear  Whenever I am alone in a dark room, walk alone...
4       shame  I bought a possible answer to a homework probl...
...       ...                                                ...
5328  sadness  After a disagreement between my parents, when ...
5329    guilt  One day I shouted at my brother who didn't do ...
5330     fear  Before one of my final exams of the third year...
5331  disgust  A drunk man bumped into me and wanted to grip ...
5332    shame  Had a very good friend. We grew apart I found ...

[5333 rows x 2 columns]

In [17]:
isear_train_df[0].value_counts() #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

joy                                                                                                                                777
guilt                                                                                                                              766
sadness                                                                                                                            760
anger                                                                                                                              758
shame                                                                                                                              757
disgust                                                                                                                            757
fear                                                                                                                               751
Note: for GUILT go to record n¦231"                    

# Data Details # Multi-class-classification
# Emotions tags:
joy, guilt, sadness, anger, shame, disgust, fear
train.csv = 5333
test.csv = 1146
val.csv = 1150